### PR TITLE
Attributes

### PR DESCRIPTION
**Pull request by domasn**
Needed to add support for `.kern`.. Seemed silly.. Adding support for them all.

### CODE REVIEW

File: TextStyle/Source/TextStylable.swift [hunk 1]
```diff
@@ -11,3 +11,21 @@ import Foundation
 public protocol TextStylable {
     func set(textStyle: TextStyle)
 }
+
+extension TextStylable {
+
+    func validateAttributes(of textStyle: TextStyle) {
+
+        if textStyle.paragraphAttributes.contains(where: { $0[\.alignment] != nil }) {
+            print("Setting `paragraphAttributes` directly on `\(type(of: self))` not supported, use `String().with(textStyle: TextStyle) -> NSAttributedString`")
```
Comment (eimantas) [inline TextStyle/Source/TextStylable.swift:20]: How about just fatalError?

**Review by eimantas — APPROVED** [review]

File: TextStyle/Source/TextStylable.swift (hunk 1 above)
Comment (domasn) [reply TextStyle/Source/TextStylable.swift:20]: Well, most of the style sill works 🤷‍♂️
In WL we have some places where style is applied to label (not sure if it should be that way). Changing it to `fatalError` might bite us bad 😬

File: TextStyle/Source/ReferenceWritableKeyPathSetter.swift [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    init<Value>(keyPath: ReferenceWritableKeyPath<Root, Value>, value: Value) {
+        self.write = { $0[keyPath: keyPath] = value }
+        self.keyPath = keyPath
+        self.value = value
+    }
+}
+
+extension Array {
+    subscript<Root, Value>(_ keyPath: ReferenceWritableKeyPath<Root, Value>) -> Value? where Element == ReferenceWritableKeyPathSetter<Root> {
+        return nil
```
Comment (domasn) [inline TextStyle/Source/ReferenceWritableKeyPathSetter.swift:22]: How did I leave this unimplemented 🤦

Comment (domasn) [reply TextStyle/Source/ReferenceWritableKeyPathSetter.swift:22]: Just did: https://github.com/trafi/TextStyle/commit/2c5439363b04aa57586af18a1f0b7a3526fdaf08

**Pull request merged**